support/src/main/java/se/inera/intyg/common/support/validate/ValidatorUtil.java

```diff
@@ -268,6 +268,17 @@ public static void addValidationErrorWithQuestionId(List<ValidationMessage> vali
         LOG.debug(field + " " + type.toString());
     }
 
+    public static void addValidationErrorWithQuestionId(List<ValidationMessage> validationMessages, String category, String field,
+        ValidationMessageType type, String msg, String dynamicLabel, String questionId) {
+
+        // Bit of a hack - but make sure no WARN types are added to the ERROR list.
+        if (type == ValidationMessageType.WARN) {
```

Comment:
Why did we need to add this in this method and not the other methods that add validation errors?

Reply:
I copied the existing method and added questionId to it. For some unknown reason when there is a dynamic label present this check is done. 

For the moment the only messageType that reaches this method is ValidationMessageType.OTHER, so removing the if-statement would not affect our code at this moment. Is it worth keeping for future error handling or should we remove it @IamFriberg?